github.go

```diff
@@ -136,22 +108,48 @@ func (importer githubImporter) getToken(i importBody) (wharfapi.Token, error) {
 	} else {
 		token, err = importer.WharfClient.GetToken(i.Token, i.User)
 		if err != nil || token.TokenID == 0 {
-			token, err = importer.WharfClient.PostToken(wharfapi.Token{Token: i.Token, UserName: i.User, ProviderID: importer.Provider.ProviderID})
+			token, err = importer.WharfClient.PutToken(wharfapi.Token{Token: i.Token, UserName: i.User, ProviderID: importer.Provider.ProviderID})
```

Comment:
Marking as "Request changes" again just as a note that https://github.com/iver-wharf/wharf-api-client-go/pull/8 needs to be merged first and then released, and then this repo's `go.mod` needs to be updated to target `wharf-api-client-go` v1.3.0